evennia/server/portal/telnet.py

```diff
@@ -49,7 +49,8 @@ def connectionMade(self):
         # this number is counted down for every handshake that completes.
         # when it reaches 0 the portal/server syncs their data
         self.handshakes = 8  # suppress-go-ahead, naws, ttype, mccp, mssp, msdp, gmcp, mxp
-        self.init_session(self.protocol_name, client_address, self.factory.sessionhandler)
+        self.init_session(self.protocol_name, client_address, self.factory.sessionhandler,
+                override_flags={"ENCODING": _TELNET_ENCODING})
```

Comment:
I don't think this `override_flags` structure is really needed? It's added only for the purpose of this one setting and won't be called by any other structure. I think it's fine to use the setting directly where it's used rather than to add this faux call.